src/search.c

```diff
@@ -170,10 +243,16 @@ static void print_final_results(void)
 		b = ptr->data;
 		ptr = ptr->next;
 		counter++;
-		printf("Bundle %s\t(%li Mb)%s\n",
-		       b->bundle_name,
-		       b->size,
-		       b->is_tracked ? "\tinstalled" : "");
+		/* do not print the size information when the scope is only one bundle ('o')
+		 * because we did not load all bundles and therefore do not have include sizes
+		 * for the result */
+		printf("Bundle %s\t%s", b->bundle_name, b->is_tracked ? "[installed]\t" : "");
+		if (scope != 'o') {
+			printf("(%li MB%s)",
+			       b->size / 1000 / 1000, /* convert from bytes->KB->MB */
```

Comment:
why 1000 and not 1024?

Reply:
it depends if we want to display sizes in `MB` of `MiB` units...

Reply:
Yes, I am reporting MB